ServiceWebsite/ServiceWebsite/ClientApp/src/app/app.component.ts

```diff
@@ -30,46 +31,34 @@ export class AppComponent implements OnInit {
 
     constructor(
         private router: Router,
-        private adalService: AdalService,
-        private config: Config,
+        private oidcSecurityService: OidcSecurityService,
         private window: WindowRef,
         private profileService: ProfileService,
         private journeySelector: JourneySelector,
         pageTracker: PageTrackerService,
         private deviceTypeService: DeviceType,
         private navigationBackSelector: NavigationBackSelector,
-        private renderer: Renderer2,
         private redirect: DocumentRedirectService,
-        private logger: Logger
+        private logger: Logger,
+        private configService: ConfigService
     ) {
         this.loggedIn = false;
-        this.initAuthentication();
         pageTracker.trackNavigation(router);
     }
 
-    private initAuthentication() {
-        const config: adal.Config = {
-            tenant: this.config.tenantId,
-            clientId: this.config.clientId,
-            postLogoutRedirectUri: this.config.postLogoutRedirectUri,
-            redirectUri: this.config.redirectUri
-        };
-
-        this.adalService.init(config);
-    }
-
     ngOnInit() {
         this.checkBrowser();
-        this.adalService.handleWindowCallback();
-        this.loggedIn = this.adalService.userInfo.authenticated;
-
-        this.initialiseProfile().then(() => (this.initialized = true));
+        this.configService.getClientSettings().subscribe(clientSettings => {
```

Comment:
do you need to subscribe to client settings here if you don't use or need it? Better to subscribe the authenticated only here.

Have a look at the documentation's sample [here](https://github.com/damienbod/angular-auth-oidc-client/blob/main/projects/sample-code-flow-http-config/src/app/app.component.ts)